src/System.Net.Primitives/src/System.Net.Primitives.csproj

```diff
@@ -124,19 +140,51 @@
     <Compile Include="$(CommonPath)\Interop\Windows\NtDll\Interop.RtlIpv6StringToAddressEx.cs" >
       <Link>Common\Interop\Windows\NtDll\Interop.RtlIpv6StringToAddressEx.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\Interop\Windows\SChannel\Interop.SchProtocols.cs" >
-      <Link>Common\Interop\Windows\SChannel\Interop.SchProtocols.cs</Link>
+  </ItemGroup>
+  
+  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' ">
+    <Compile Include="System\Net\IPAddressPal.Unix.cs" />
```

Comment:
@pgavlin Reading the x-plat guidelines and the System.IO implementation I see that the requirement is to rather have partial classes that provide the specifics for each OS: IPAddress.Unix.cs instead of a PAL layer as we have thought of initially.

Unfortunately I didn't have enough time to think through the advantages/disadvantages. Do you have arguments pro/against the recommended pattern?